shimmingtoolbox/cli/mask.py

```diff
@@ -0,0 +1,153 @@
+                                                                             "and third dimension (in pixels). If no "
+                                                                             "center is provided, the middle is used.")
+def cube(fname_input, output, size, center):
+    """
+        Create a cube mask from the input file. Return an output nifti file with cube mask.
+
+        Args:
+            fname_input (str): Complete input path of the nifti file to mask.
+            output (str): Output folder for cube mask.
+            size (int): Length of the side of the cube along first, second and third dimension (in pixels).
+            center (int): Center of the cube along first, second and third dimension (in pixels). If no center is
+                            provided, the middle is used.
+            
+        Return:
+            output (str): Filename for the output mask.
+        """
+    # Create the folder where the nifti file will be stored
+    if not os.path.exists(output):
+        os.makedirs(output)
+
+    nii = nib.load(fname_input)
+    data = nii.get_fdata()  # convert nifti file to numpy array
+
+    if len(data.shape) == 3:
+        mask_cb = shape_cube(data, size, size, size, center[0], center[1], center[2])  # creation
+        # of the cube mask
+        mask_cb = mask_cb.astype(int)
+        nii_img = nib.Nifti1Image(mask_cb, nii.affine)
+        fname_mask = os.path.join(output, 'mask.nii.gz')
+        nib.save(nii_img, fname_mask)
+        click.echo(f"The filename for the output mask is: {os.path.abspath(fname_mask)}")
+        return output
+
+    else:
+        raise ValueError("The nifti file does not have 3 dimensions.")
+
+
+@mask_cli.command(context_settings=CONTEXT_SETTINGS,
+              help=f"Create a square mask from the input file. Return an output nifti file with square mask.")
+@click.option('-input', 'fname_input', type=click.Path(), required=True, help="Input path of the nifti file to mask")
+@click.option('-output', type=click.Path(), default=os.curdir, help="Output folder for mask in nifti file")
+@click.option("-size", required=True, type=int, help="Length of the side of the square along first and second dimension"
+                                                     " (in pixels)")
+@click.option("-center", nargs=2, type=int, default=(None, None), help="Center of the cube along first and second "
+                                                                       "dimension (in pixels). If no center is "
+                                                                       "provided, the middle is used.")
+def square(fname_input, output, size, center):
+    """
+            Create a square mask from the input file. Return an output nifti file with square mask.
+
+            Args:
+                fname_input (str): Complete input path of the nifti file to mask.
+                output (str): Output folder for square mask.
```

Comment:
i thought that with the new sphinx_click, these three-quotes were no more required?

Reply:
True, it's useless. I will remove it and be more precise in my decorators for a better documentation.